### PR TITLE
feat: [PLG-2419]: Added telemetry and K8s swimlanes

### PR DESCRIPTION
**Pull request by devinder-harness**
### Summary
Added:

Telemtry
K8s Swimlanes
Kustomize
Helm with forks
Serverless Swimlanes (Disabled in this PR)
Links to doc
Changed installation instructions for Mac and Linux
JIRA : https://harness.atlassian.net/browse/PLG-2419

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cd/strings/strings.en.yaml [hunk 1]
```diff
@@ -684,29 +695,33 @@ getStartedWithCD:
           description: In the GitOps Workflow approach, Harness GitOps will watch the state of your application as defined in a git repo and will then pull any changes into the Kubernetes cluster. You have the Hosted GitOps and Self-Managed GitOps options to get started with Harness GitOps.
     deploymentSteps:
       headsteps:
-        preparation: '1. Preparation'
-        createEntities: '2. Create the required Harness entities'
-        verifyPipeline: '4. Verify that pipeline has been created'
-      createEntitytitle: Run these commands to create the Harness entities necessary to deploy your pipeline. Learn more
+        preparation: '1.Download CLI and set up GitHub access'
```
Comment (abhinavrastogi-harness) [inline src/modules/75-cd/strings/strings.en.yaml:698]: spaces after list number

File: src/modules/75-cd/strings/strings.en.yaml [hunk 2]
```diff
@@ -684,29 +695,33 @@ getStartedWithCD:
...
+        createEntities: '{{num}}. Create the required Harness entities'
+        verifyPipeline: '{{num}}. Verify that pipeline has been created'
+        configureGcp: '2.Configure GCP'
+        configureAws: '2.Configure AWS'
+      createEntitytitle: 'Run these commands to create the Harness entities necessary to deploy your pipeline. <a href="https://github.com/harness/harness-cli" target="_blank">Learn more </a>'
```
Comment (abhinavrastogi-harness) [inline src/modules/75-cd/strings/strings.en.yaml:703]: we need to add `rel='nofollow noreferrer'`

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by sahithikolichala — APPROVED** [review]

**Comment by devinder-harness** [comment]
tracking covergae here https://harness.atlassian.net/browse/PLG-2652

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/ApiKeySetup.tsx [hunk 3]
```diff
@@ -111,75 +117,70 @@ export default function ApiKeySetup({
   }
 
   const createApiToken = async (apikeyid = apiKey): Promise<void> => {
+    const validTo = moment().add('30', 'd').unix() * 1000
```
[the remark this answers is not in the document]
Comment (devinder-harness) [reply src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/ApiKeySetup.tsx:120]: https://harness.atlassian.net/browse/PL-40654

File: src/modules/75-cd/strings/strings.en.yaml (hunk 1 above)
Comment (devinder-harness) [reply src/modules/75-cd/strings/strings.en.yaml:698]: These changes will be part of the next PR

File: src/modules/75-cd/strings/strings.en.yaml (hunk 2 above)
Comment (devinder-harness) [reply src/modules/75-cd/strings/strings.en.yaml:703]: These changes will be part of the next PR

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/AWS/ConfigureAWS.tsx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+          {getString('cd.getStartedWithCD.flowByQuestions.deploymentSteps.steps.configureAWSStep.awsBucketName')}
+        </Label>
+        <TextInput
+          id="gcpbucket"
+          name="gcpbucket"
+          defaultValue={''}
+          placeholder={getString(
+            'cd.getStartedWithCD.flowByQuestions.deploymentSteps.steps.configureGCPStep.placholders.bucketName'
+          )}
+          // onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
```
[the remark this answers is not in the document]
Comment (devinder-harness) [reply src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/DeploymentSetupSteps/AWS/ConfigureAWS.tsx:113]: These changes will be part of the next PR..being implemented as WIP feature

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Constants.ts [hunk 5]
```diff
@@ -91,9 +164,37 @@ export const DEPLOYMENT_STRATEGY_TYPES: {
...
+      {
+        title: 'Step 3',
+        description: 'pipeline.executionStrategy.strategies.rolling.steps.step3.description'
+      }
+    ],
+    pipelineCommand:
+      'cd.getStartedWithCD.flowByQuestions.deploymentSteps.steps.pipelineSetupStep.commands.createpipeline.createrollingcmd'
   }
 }
+export const DEPLOYMENT_TYPE_MAP: { [key: string]: string[] } = {
```
[the remark this answers is not in the document]
Comment (devinder-harness) [reply src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Constants.ts:188]: These changes will be part of the next PR

File: src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Constants.ts [hunk 6]
```diff
@@ -4,19 +4,32 @@
...
 import CanaryVideo from '@pipeline/components/PipelineStudio/ExecutionStrategy/resources/Canary-deployment.mp4'
 import RollingUpdateVideo from '@pipeline/components/PipelineStudio/ExecutionStrategy/resources/Rolling-Update-deployment.mp4'
+import HelmDeploymentBasic from '@pipeline/components/PipelineStudio/ExecutionStrategy/resources/Helm-Deployment-basic.mp4'
 import { KubernetesType } from '@delegates/constants'
 import type { EntityMap, DeploymentStrategyTypes, DeploymentFlowType } from './types'
-
+import css from './CDOnboardingWizardWithCLI.module.scss'
+interface KVPair {
+  [key: string]: string
+}
```
[the remark this answers is not in the document]
Comment (devinder-harness) [reply src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Constants.ts:17]: These changes will be part of the next PR

**Pull request merged**